### PR TITLE
DEVPROD-3295 update volume size default

### PR DESCRIPTION
**Pull request by ablack12**
DEVPROD-3295
### Description
500 GB is supposed to be the max volume size that users can request, not the default that we suggest. Infra suggested that we start recommending 256 as a power of two.

### CODE REVIEW

File: src/components/Spawn/spawnHostModal/getFormSchema.tsx [hunk 1]
```diff
@@ -408,7 +408,7 @@ export const getFormSchema = ({
                       volumeSize: {
                         title: "Volume size (GB)",
                         type: "number" as "number",
-                        default: 500,
```
Comment (sophstad) [inline src/components/Spawn/spawnHostModal/getFormSchema.tsx:411]: Would prefer to update this constant and then use that variable in place of this magic number! https://github.com/evergreen-ci/spruce/blob/de2fac705d2400ea62c68708a3ee8ebbc4b18a98/src/components/Spawn/spawnHostModal/constants.ts#L1

**Review by sophstad — CHANGES_REQUESTED** [review]
I think a test in `src/pages/spawn/spawnVolume/SpawnVolumeModal.test.tsx` may need updating as well

File: src/components/Spawn/spawnHostModal/getFormSchema.tsx (hunk 1 above)
Comment (ablack12) [reply src/components/Spawn/spawnHostModal/getFormSchema.tsx:411]: perf thanks!

**Comment by ablack12** [comment]
I'm not sure what to look at for the cypress/flake, can you let me know if there's something to be done here?

**Review by sophstad — APPROVED** [review]
LGTM! No worries about the flake

**Pull request merged**